components/AlertTItle/AlertTitle.interface.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+export interface IAlertTitle {
+  className?: string;
```

Comment:
I don't know if I commented about this in your PR, but it was somewhere. There should be no use of classname. This should be a set standard classes. If there's a condition, it should be handled inside the component.